var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+
+
+class Zee(CMakePackage):
+
+    """Zee is a collection of scalable FEM miniapps based on Omega_h providing
+    the basis for an efficient and scalable SSA + E-field client."""
+
+    homepage = "https://github.com/BlueBrain/zee"
+    url      = "git@github.com:BlueBrain/zee.git"
+
+    version('develop', git=url, submodules=True)
+
+    variant('debug', default=False, description='Build debug with O0')
+    variant('build_type', default=' ', description='CMake build type',
+            values=' ')
+    depends_on('cmake@3:', type='build')
+    depends_on('pkg-config', type='build')
+    depends_on('boost')
+    depends_on('gmsh -mpi %gcc')
+    depends_on('mpi')
+    depends_on('omega-h')
+    depends_on('petsc +int64')
```

Comment:
if above changes to petsc are to be taken into accout, don't we need the variant for `patchmpi64 ` here?

Reply:
For now I want to apply the patch on demand, so that I can test easily other MPI implementations without.